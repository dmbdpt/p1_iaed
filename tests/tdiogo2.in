t 1 abc
t 2 abcde
n 1
u pedro
t 3 meh
m 3 pedro IN PROGRESS
d TO DO
d IN PROGRESS
n 1
m 2 pedro IN PROGRESS
d TO DO
d IN PROGRESS
q
