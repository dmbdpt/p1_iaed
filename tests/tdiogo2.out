task 1
task 2
1
task 3
1 0 abc
2 0 abcd
3 1 meh
2
1 0 abc
3 1 meh
2 2 abcd
